website/server/controllers/api-v3/user.js

```diff
@@ -223,6 +223,10 @@ api.deleteUser = {
 
     await user.remove();
 
+    if (req.body.feedback) {
+      // Send feedback via email
```

Comment:
For reference, this is what needs to be added: https://github.com/HabitRPG/habitica/blob/develop/website/server/controllers/api-v3/chat.js#L266 . I'll add it if this is updated soon.